src/components/fieldset/Fieldset.js

```diff
@@ -40,4 +41,11 @@ export default class FieldsetComponent extends NestedComponent {
     super(...args);
     this.noField = true;
   }
+
+  createComponent(component, options, data, before) {
+    const comp = super.createComponent(component, options, data, before);
+    if (document.querySelectorAll(`[id^=${comp.id}]`).length) {
```

Comment:
This seems really strange. Why didn't createComponent add the id in the first place? Why do we need to add a querySelectorAll here?

Reply:
removed querySelectorAll